### PR TITLE
Fix uniqueCount method #263 & Add arrayFill method

### LINKED ISSUE #263: Fix uniqueCount function
There is currently a method called `uniqueCount` that lives in `/lib/uniqueCount.js`.
It is incomplete and needs to be fixed!

### Acceptance Criteria:
1. Running `yarn test uniqueCount` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by amoradi**
Closes #263  

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: test/arrayFill.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const { arrayFill } = require("../lib");
+
+describe("arrayFill", () => {
+  test("should return a new array", () => {
```
Comment (BlakeGuilloud) [inline test/arrayFill.test.js]: Hey @amoradi !! Thanks so much for the contrib. We really appreciate it. So I believe the `.toBe` methods here should actually be `.toEqual`:

```javascript
  test("some stuff", () => {
    var foo = [1, 6, 7, "cake"];
    var bar = [1, 6, 7, "cake"];

    expect(bar).not.toBe(foo);
  });
```

The above test will pass, even though it should not.

```javascript
  test("should return a new array", () => {
    var foo = [1, 6, 7, "cake"];
    var bar = [1, 6, 7, "cake"];

    expect(bar).not.toEqual(foo);
  });
```

However ^^ This test will fail with the return of:

```
    Expected value to not equal:
      [1, 6, 7, "cake"]
```

I love the method tho! Nice work
